ci/gpu/build.sh

```diff
@@ -83,8 +83,8 @@ gpuci_conda_retry install -y \
                   "ucx-py=${MINOR_VERSION}"
 
 # https://docs.rapids.ai/maintainers/depmgmt/
-# gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
-# gpuci_conda_retry install -y "your-pkg=1.0.0"
+gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry install -y "dlpack=0.5"
```

Comment:
Looks like the [failing jobs](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci-v0.20/job/cudf/job/prb/job/cudf-cpu-cuda-build/CUDA=11.2/1033/consoleText) are using `ci/cpu/build.sh`, so I suspect we also need these two lines [here](https://github.com/rapidsai/cudf/blob/branch-21.06/ci/cpu/build.sh#L54).